W/WhereAmI/build_tarballs.jl

```diff
@@ -0,0 +1,32 @@
+using BinaryBuilder
+
+name = "WhereAmI"
+version = v"2024.8.6"  # no proper versions/releases, this is the date of the commit used
```

Comment:
For lack of proper veesioning, how about something like 0.1.0 or 1.0.0? Calendar-based veesioning plays very badly with semver

Reply:
Sure, makes sense
```suggestion
version = v"1.0.0"  # no proper versions/releases, this is made up
```